test/src/unit/modules/User.test.js

```diff
@@ -314,7 +317,7 @@ describe('User.js', () => {
 
             await user.cancelLoad(project)
                 .should.eventually.be.rejectedWith('Load Runner service is not available');
-        });
+        }).timeout(testTimeout.short);
```

Comment:
Nit: Maybe set the whole of User.js to use the short testTimeout?
```
describe('User.js', function() { 
    // Set the default timeout for all tests
    this.timeout(testTimeout.short);
```

Reply:
Nah. Default test timeout is 2000ms, which wasn't enough for that particular test but fine for all the others. testTimeout.short is 20000ms, so that would be a whole degree of magnitude more to wait if any of the other tests timeout. Keeping the timeouts short is best, failing faster so as to free up Jenkins resources.

Reply:
Thats fine. Can resolve this.